README.md

```diff
@@ -1,5 +1,48 @@
+**![](https://img.shields.io/badge/-microverse-yellow)**
+
 # Basic HTML Page Structure (Youtube Page)
 
 > This is the first project on CSS/HTML curriculum in microverse course
 
+# Screenshot
+
+![screenshot](/src/screenshot_1.png)
+
+## Participants
```

Comment:
On the microverse readme template, this title is `Author(s)`, if you want to customize it I would suggest using like `developers` or `contributors` this make more sense than just participant. I'd suggest you to work on this as well.